lib/mapObjects/CBank.h

```diff
@@ -22,6 +22,7 @@ class DLL_LINKAGE CBank : public CArmedInstance
 	ui32 daycounter;
 	ui32 resetDuration;
 	bool coastVisitable;
+	bool regularUnitPlacement;
```

Comment:
This might be a problem for a hotfix, not sure how to solve it nicely.

It needs to be serialized, as everything in CBank.
You can add check for save version and it would work for saves.

But in multiplayer we don't have save compatibility checks. Meaning, 1.5.1 client won't be able to deserialize gamestate from 1.5.2 server (and vice versa).

Reply:
I think I have an idea how to handle multiplayer.

So this PR should be OK for 1.5.2